spec/controllers/application_controller/automate_spec.rb

```diff
@@ -1,7 +1,7 @@
 describe MiqAeCustomizationController, "ApplicationController::Automate" do
   context "#resolve" do
     before(:each) do
-      set_user_privileges
+      stub_user(features: :all)
```

Comment:
In all honesty though, these should all be hash rockets...from the caller side, it's a regular hash, not kwargs.  kwargs syntax at the signature level, which is what it was designed for, is required: `def stub_user(features:)` is required syntax (vs `def stub_user(:features =>)` which is invalid syntax)